Formula/r/rqbit.rb

```diff
@@ -0,0 +1,54 @@
+class Rqbit < Formula
+  desc "Fast command-line bittorrent client and server"
+  homepage "https://github.com/ikatson/rqbit"
+  url "https://github.com/ikatson/rqbit/archive/refs/tags/v8.1.1.tar.gz"
+  sha256 "452b8260fabba938567e1819a9edfcf6b69579ecd5f8b87fee4ca1666fa8fede"
+  license "Apache-2.0"
+  head "https://github.com/ikatson/rqbit.git", branch: "main"
+
+  depends_on "node" => :build
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "openssl@3"
+    depends_on "pkgconf"
+  end
+
+  def install
+    if OS.linux?
+      # Ensure the declared `openssl@3` dependency will be picked up.
+      # https://docs.rs/openssl/latest/openssl/#manual
+      ENV["OPENSSL_DIR"] = Formula["openssl@3"].opt_prefix
+      ENV["OPENSSL_NO_VENDOR"] = "1"
+    end
+
+    system "cargo", "install", *std_cargo_args(path: "crates/rqbit")
+
+    generate_completions_from_executable(bin/"rqbit", "completions", shells: [:bash, :fish, :zsh])
```

Comment:
Default shells are equal to this.
```suggestion
    generate_completions_from_executable(bin/"rqbit")
```